suite/tests/CMakeLists.txt

```diff
@@ -2776,40 +2776,41 @@ if (CLIENT_INTERFACE)
     endif ()
   endif (ARM)
 
-  if (NOT ARM AND NOT AARCH64) # FIXME i#1578: fix detach on ARM/AArch64
+  if (NOT AARCHXX) # TODO i#1578: Crashing on AArch64.
```

Comment:
Is the crash mentioned here related to the `XXX i#1578` comment in `core/unix/os.c`?

Reply:
Oh that comment is just for pure-asm apps. So probably not.

Reply:
The os.c comment about no-libc apps?  No, that was affecting the allasm tests which have no TLS of their own b/c they don't use any libraries.